test/bats/helpers.bash

```diff
@@ -111,3 +111,10 @@ wait_for_process() {
   done
   return 1
 }
+
+update_crl_server() {
```

Comment:
a more description name would be, revokeCert

Reply:
Sure